cloudvolume/cloudvolume.py

```diff
@@ -160,6 +157,27 @@ def __init__(self, cloudpath, mip=0, bounded=True, autocrop=False, fill_missing=
 
     self.pid = os.getpid()
 
+  @classmethod
+  def from_numpy(cls, arr, vol_path='file://tmp/'+generate_random_string(),
+                  resolution=(4,4,40), voxel_offset=(0,0,0), 
+                  chunk_size=(64,64,64)):
+    mkdir(vol_path)
```

Comment:
This only applies to the `file://` protocol.

Reply:
I see, I have to use `/tmp/image/dfds` rather than `/tmp/dfds`